<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>711</width>
    <height>531</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>485</width>
    <height>412</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>华文隶书</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>智能化中药房管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Lenovo/.designer/res_img/中药.png</normaloff>C:/Users/Lenovo/.designer/res_img/中药.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>601</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane{
min-width:70px;
min-height:25px;
border-top: 2px ;
}
QTabWidget::tab-bar{padding-top:40px;}
QTabBar::tab {
min-width: 140px;
margin: 10px; 

min-width:70px;

min-height:25px;

color: rgb(50,50,50);

font:28px &quot;华文行楷&quot;;

border: 0px solid;



}

QTabBar::tab:selected{

min-width:70px;

min-height:25px;
color: rgb(0,0,0);

font:30px &quot;华文行楷&quot;;

border: 0px solid;

border-bottom: 2px solid;

border-color: black;

}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>129</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   background-color:rgba(0,0,0,0);
   border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
   
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能化中药房管理系统</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>219</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_manager">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>400</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>管理员登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 0);
    color: black;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>记住</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>340</y>
       <width>93</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_75">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color:rgba(0,0,0,0);
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
     <property name="placeholderText">
      <string>姓名</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255,80);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Lenovo/.designer/res_img/山水.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
